### PR TITLE
Sub Category issues

### PR DESCRIPTION
**Pull request by akathuria289**
## Issue Reference
#7330 #7331 #7361

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
**Not fixed**, still not able to **click** on **sub-subcategory**, from the categories **drop-down**.

**Comment by akathuria289** [comment]
@Sanjaybhattwebkul Fixed

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
**Not working** if there are 3 sub-sub category Like **parent->_child_->**child**->**child****
In this case, the last child category is **not visible** on the child dropdown.

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
Any update?

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
Resolve the **conflicts**.

File: config/db-blade-compiler.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	'model_property' => '__db_blade_compiler_content_field',
+
+	/**
+	 * The default model field to be compiled when not explicitly specified
+	 * with DbView::field()
+	 */
+	'model_default_field' => 'content',
+
+    'cache' => false
+);
```
Comment (devansh-webkul) [inline config/db-blade-compiler.php:19]: Remove vendor folder and run composer install.

Comment (devansh-webkul) [reply config/db-blade-compiler.php:19]: This file should not be pushed.

Comment (akathuria289) [reply config/db-blade-compiler.php]: removed

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
Resolve the **conflicts**.

**Comment by akathuria289** [comment]
@Sanjaybhattwebkul Resolved

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**